NEWS.md

```diff
@@ -1,5 +1,8 @@
 # pkgdown (development version)
 
+* pkgdown now supports local searching. It is enabled by default because no set-up is needed for users to
```

Comment:
Should we comment on the fact that the sitemap will get is also more complete now?

Reply:
Sure, although I don't think many people will care/notice.